client/features/auth/CreateAccount.jsx

```diff
@@ -97,13 +97,13 @@ class CreateAccount extends React.Component {
             <input autoFocus type="text" name="fname" id="name" placeholder="Name" onChange={this.handleChange}></input>
           </div>
           <div className="ca-username">
-            <input type="text" name="username" autocomplete="username" id="username" placeholder="Username" onChange={this.handleChange}></input>
+            <input type="text" name="username" autoComplete="username" id="username" placeholder="Username" onChange={this.handleChange}></input>
           </div>
           <div className="ca-email">
-            <input type="text" name="email" autocomplete="email" id="email" placeholder="Email" onChange={this.handleChange}></input>
+            <input type="text" name="email" autoComplete="email" id="email" placeholder="Email" onChange={this.handleChange}></input>
           </div>
           <div className="ca-password">
-            <input type="password" name="password" autocomplete="new-password" id="password" placeholder="Password" onChange={this.handleChange}></input>
+            <input type="password" name="password" autoComplete="new-password" id="password" placeholder="Password" onChange={this.handleChange}></input>
           </div>
           <div className="ca-subscriptionOptions">
             <label>Select Your Subscriptions</label>
```

Comment:
Make sure to include hulu, netflix, amazon prime video, apple tv plus, disney plus, and hbo max.

Reply:
I think a lot of this is because I've been merging other branches to keep from causing conflicts.